teacher/templates/teacher.html

```diff
@@ -1,98 +1,89 @@
-      <h2> Announcements </h2><hr>
-      {% for announcement in announcements reversed %}
-            <div class="announcement">
-            <h4>{{ announcement.title }}</h4><hr>
-            <p>{{ announcement.announcement }}</p><hr>
-            <p>{{ announcement.created_by }} on {{ announcement.posted.date }}</p><br>
-            <form method="POST">
-              {% csrf_token %}
-              <input type="hidden" name="dismiss_announcement" value="true" />
-              <input type="hidden" name="announcement" value="{{ announcement.id }}" />
-              <button class="button">Dismiss</button><br><br>
-            </form>
-            </div><br>
-      {% endfor %}
+      <div style="text-align: center">
+        <div style="width: 50%; display: inline-block" class="mui-panel">
+                <h3> Announcements </h3><hr>
+        {% for announcement in announcements reversed %}
+              <div class="announcement">
+              <h2>{{ announcement.announcement.title }}</h2><hr>
+              <p>{{ announcement.announcement.announcement }}</p><br>
+              <p>{{ announcement.announcement.created_by }} on {{ announcement.announcement.posted.date }}</p><hr/>
+              <br>
+              <form method="POST">
+                {% csrf_token %}
+                  <p style="padding-bottom: 10px;">
+                      <button class="mui-btn">Dismiss</button>
+                  </p>
+                <input type="hidden" name="dismiss_announcement" value="true" />
+                <input type="hidden" name="announcement" value="{{ announcement.announcement.id }}" />
+              </form>
+              </div><br>
+        {% endfor %}
+        </div>
+      </div>
     {% endif %}
     {% if forms %}
-      <h2> Forms </h2><hr>
-        {% for form in forms %}
+    <div style="text-align: center">
+      <div style="width: 50%; display: inline-block" class="mui-panel">
+      <h3> Forms </h3><hr>
+        {% for form in forms reversed %}
             <div class="form">
-              <h4>{{ form.name }}</h4><hr>
-              <p>{{ form.description }}</p><br>
-              <p>{{ form.created_by }} on {{ form.posted.date }}</p><br>
-              <p>
-                <form action="{% url 'download_form' %}" method="get">
-                <input type="hidden" name="path" value="{{ form.form }}">
-                <button class="button">Download Form</button><br>
-                </form>
+              <h2>{{ form.form.name }}</h2><hr>
+              <p>{{ form.form.description }}</p><br>
+              <p>{{ form.form.created_by }} on {{ form.form.posted.date }}</p><br><hr>
+              {% if form.form.esign != NULL %}
+                <p><a href="{{ form.form.esign.template }}">Esign Here</a></p><br>
+              {% endif %}
+              <p style="padding-bottom: 10px;">
+                  <input type="button" class="mui-btn" value="Download Form" onclick="window.location.href='{{ form.form.url }}'"/>
```

Comment:
Should be a link, no javascript

Reply:
Resolved 6eb5cb979b3e7f5677a52dfe3d696b44155aa78e